src/components/PageHeader.tsx

```diff
@@ -52,11 +51,39 @@ export const PageHeader = ({ backPath, children, ...props }: PageHeaderProps) =>
   );
 };
 
+const StyledButton = styled(Button)`
+  background-color: ${({ theme }) => theme.palette.section.light};
+  color: ${({ theme }) => theme.palette.section.megaDark};
+  padding: 0.2rem;
+  min-width: 0;
+`;
+
 const StyledRegistrationPageHeader = styled(PageHeader)`
   font-weight: 700;
   font-style: italic;
 `;
 
-export const RegistrationPageHeader = (props: PageHeaderProps) => (
-  <StyledRegistrationPageHeader variant="h2" variantMapping={{ h2: 'h1' }} {...props} />
-);
+export const RegistrationPageHeader = (props: { children: string }) => {
+  const [showFullText, setShowFullText] = useState(false);
+
+  const toggleFullText = () => setShowFullText(!showFullText);
+
+  return (
+    <StyledRegistrationPageHeader variant="h2" variantMapping={{ h2: 'h1' }} onClick={toggleFullText} {...props}>
+      {showFullText ? (
```

Comment:
burde ikke logikken for show all osv ligge i `<PageHeader>`? tenker den er like relevant for feks "Personverneklæring" og andre lange headers på mobil

Reply:
jo, kan godt flytte det dit 👍